tests/test_data.py

```
@@ -719,6 +719,89 @@ def test_data_indexerror_on_missing_index(self):
             _ = data[0]
 
 
+class TestUnbinnedDataWeightedEntries:
+    """Tests for UnbinnedData.weighted_entries property."""
+
+    def _make_1d(self, entries, weights=None):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_make_1d`

(ANN202)